src/zui/ZUITooltip/index.tsx

```diff
@@ -1,30 +1,35 @@
-import { Tooltip, TooltipProps } from '@mui/material';
+import { Tooltip } from '@mui/material';
 import { FC, ReactElement } from 'react';
 
-export interface ZUITooltipProps {
+type ZUITooltipProps = {
+  /** If the tooltip should have a small arrow or not. Defaults to 'true' */
+  arrow?: boolean;
+
   children: ReactElement;
-  arrow?: 'None' | 'Up' | 'Down' | 'Left' | 'Right';
+
+  /** The text content of the tooltip */
   label: string;
-}
 
-const ZUITooltip: FC<ZUITooltipProps> = ({ children, arrow, label }) => {
-  let showArrow = true;
-  let placement: TooltipProps['placement'] = 'top';
-  if (arrow == 'None') {
-    showArrow = false;
-    placement = 'top';
-  } else if (arrow == 'Up') {
-    placement = 'bottom';
-  } else if (arrow == 'Down') {
-    placement = 'top';
-  } else if (arrow == 'Left') {
-    placement = 'right';
-  } else if (arrow == 'Right') {
-    placement = 'left';
-  }
+  /** Where the tooltip should attach in relation to its children.
+   *
```

Comment:
I like to reserve the first line of a documentation block comment so that there's no text there.

```suggestion
  /** 
   * Where the tooltip should attach in relation to its children.
   *
```